README.md

```diff
@@ -1,6 +1,6 @@
-# Getting Started with Okteto Stacks
+# Getting Started with Docker Compose
 
-This example shows how to leverage [Okteto](https://github.com/okteto/okteto) to develop an application using [Stacks](hhttps://okteto.com/docs/cloud/stacks#manifest-reference). Stacks are designed for developers that don't want to deal with the complexities of Kubernetes manifests or Helm charts.
+This example shows how to leverage [Okteto](https://github.com/okteto/okteto) to develop an application using the [Compose spec](https://github.com/compose-spec/compose-spec). Compose is designed for developers that don't want to deal with the complexities of Kubernetes manifests or Helm charts.
```

Comment:
I don't think the "compose is design for developers..." part makes sense with this approach. Maybe mention that okteto gives you the option to use compose or something like that?

Reply:
Also, I'd say "docker compose file" instead of "compose spec" since that how most people think of it (to me, the spec is something that implementors care about)